scripts/nginx/mylar.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Nginx Configuration for Mylar
+# Author: Public920
+
+user=$(cut -d: -f1 < /root/.master.info)
+
+isactive=$(systemctl is-active mylar)
+if [[ $isactive == "active" ]]; then
+    systemctl stop mylar
```

Comment:
Is this necessary if there is no configuration being made to mylar itself? Might be a remnant of when the config was still here.